runtime-modules/argo-bridge/src/lib.rs

```diff
@@ -137,6 +137,33 @@ decl_module! {
             Ok(())
         }
 
+        // TODO: add weight for revert_outbound_transfer
+        #[weight = WeightInfoArgo::<T>::finalize_inbound_transfer()]
+        pub fn revert_outbound_transfer(
+            origin,
+            transfer_id: TransferId,
+            revert_account: T::AccountId,
+            revert_amount: BalanceOf<T>,
+            rationale: vec::Vec<u8>,
+        ) -> DispatchResult {
+            ensure!(Self::operator_account().is_some(), Error::<T>::OperatorAccountNotSet);
+            let caller = ensure_signed(origin)?;
+            ensure!(caller == Self::operator_account().unwrap(), Error::<T>::NotOperatorAccount);
+
+            ensure!(Self::status() == BridgeStatus::Active, Error::<T>::BridgeNotActive);
```

Comment:
Should this be present?
I am thinking about the following scenario:
1. Transaction outbound exists
2. Bridge is paused
3. User tries to revert transaction

Reply:
Yes, when the bridge is paused, nothing should be callable. Also it's not user that reverts the transaction, it's the operator